packages/components/src/KaizenProvider/KaizenProvider.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { KaizenProvider, KaizenProviderProps } from "./KaizenProvider"
+
+const KaizenProviderWrapper = (
+  props?: Partial<KaizenProviderProps>
+): JSX.Element => <KaizenProvider>{props?.children}</KaizenProvider>
+
+describe("<KaizenProvider />", () => {
+  it("does something", () => {
```

Comment:
I feel we can do a bit better than this ;)

Reply:
Also missing any tests for `ThemeManager` unless I missed them.

Reply:
Ah yeah, forgot to move the `ThemeManager` test up here.

Reply:
> I feel we can do a bit better than this ;)
Haha yeah 😅 only real job is to render children, so updated test to be more accurate.